src/main/java/edu/greenblitz/tobyDetermined/commands/telescopicArm/extender/DebugGrowBy5cm.java

```diff
@@ -0,0 +1,11 @@
+package edu.greenblitz.tobyDetermined.commands.telescopicArm.extender;
+
+import edu.greenblitz.tobyDetermined.RobotMap;
+import edu.greenblitz.tobyDetermined.subsystems.telescopicArm.Extender;
+
+public class DebugGrowBy5cm extends ExtendToLength {
+
+	public DebugGrowBy5cm() {
```

Comment:
Same as angle one, and its not grow - it can be in either direction

Reply:
@Tal you just wanted to use the word "grow" didn't you?

Reply:
amir wrote this